_data/i18n/en.yml

```diff
@@ -316,3 +317,14 @@ script:
     dependents: Dependents
     dependencies: Dependencies
     usage: Usage
+    last_commit_alt: Last commit icon
```

Comment:
I don't think these alts are very useful to hear. Isn't it better to hear nothing than to hear "bla bla icon"?